run/system_package/go.mod

```diff
@@ -1,3 +1,7 @@
-module graphviz-web
+module github.com/GoogleCloudPlatform/golang-samples/graphviz-web
 
 go 1.12
+
+require github.com/GoogleCloudPlatform/golang-samples v0.0.0
+
+replace github.com/GoogleCloudPlatform/golang-samples => ../../
```

Comment:
Add a TODO to remove this? Does this work in deploys?

Reply:
I'm currently caught in a place where I can run tests or build the container, not both. It might be more effective to complete the PR review in the context of the sample changes, then before final merge remove the sample and do a fast-follow with clean ups.